sdk/storage/storage-blob/src/pollers/BlobStartCopyFromUrlPoller.ts

```diff
@@ -59,7 +60,14 @@ export interface BlobBeginCopyFromUrlPollState
  * @ignore
  */
 export interface BlobBeginCopyFromURLPollOperation
-  extends PollOperation<BlobBeginCopyFromUrlPollState, BlobBeginCopyFromURLResponse> {}
+  extends PollOperation<
+    BlobBeginCopyFromUrlPollState,
+    BlobBeginCopyFromURLResponse,
+    | BlobStartCopyFromURLResponse
```

Comment:
maybe create an alias for all the types of the partial response and then use that instead of in-lining it?